packages/vuetify/src/components/VTabs/_variables.scss

```diff
@@ -3,6 +3,7 @@
 $tabs-bar-height: 48px !default;
 $tabs-icons-and-text-bar-height: 72px !default;
 $tabs-item-align-with-title-margin: 42px !default;
+$tabs-item-letter-spacing: .0892857143em !default;
```

Comment:
very magic

Reply:
![image](https://user-images.githubusercontent.com/9064066/69550285-5b399980-0f60-11ea-8334-6772bea99355.png)

Reply:
This is from the Material Design site demo. https://material.io/components/tabs/